src/plugins/workspace/server/saved_objects/integration_tests/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -125,14 +131,19 @@ describe('WorkspaceSavedObjectsClientWrapper', () => {
     jest.spyOn(utilsExports, 'getPrincipalsFromRequest').mockImplementation((request) => {
       if (request === notPermittedRequest) {
         return { users: ['bar'] };
+      } else if (request === permittedRequest) {
+        return { users: ['foo'] };
       }
-      return { users: ['foo'] };
+      return { groups: ['dashboard_admin'] };
```

Comment:
Can we add some test with only user id matched?

Reply:
Yes, I will add some new test with only user id matched.